charts/lnd/templates/lnd-pass-secret.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: {{ printf "%s-pass" (include "lnd.fullname" .) }}
   labels:
     {{- include "lnd.labels" . | nindent 4 }}
-stringData:
-  password: {{ default "randAlphaNum 24" .Values.walletPassword | quote }}
+data:
+  password: {{ template "walletPassword"  .  }}
```

Comment:
it's already in base64?

Reply:
The templates yes. I had missed adding the password from values.yaml so I have fixed that now.